src/desktop/components/search_bar/view.coffee

```diff
@@ -85,6 +88,35 @@ module.exports = class SearchBarView extends Backbone.View
     @renderFeedback()
     @$el.addClass 'is-loading'
 
+  shouldMeasurePerformance: ->
+    performance? and sd.VOLLEY_ENDPOINT
```

Comment:
Is this `window.performance`? What is that, exactly?

Reply:
Yep! Seems to be well supported in browsers today: https://developer.mozilla.org/en-US/docs/Web/API/Performance